### DISCUSSION TITLE
Pull up the Numeric Keypad when a Numeric field is selected

### DISCUSSION THREAD

**Discussion by bdeboer-x** [victorbalssa/abacus#174] [Ideas]
Can you make it so when you are expected to enter in a numeric value, that you pull up the numeric keypad, and not the regular keypad?  When I hit new transaction, and enter in the amount, it pulls up the regular keypad, which is a bit of a pain when entering in a value

**Comment by victorbalssa** [comment]
Thanks @bdeboer-x, I created a Feature Request from this discussion, good idea.

**Reply by victorbalssa** [reply]
@bdeboer-x this was updated in `0.10.2` let me know if you have any issues with new keyboard.
